packages/ui/src/containers/AuthContainer.ts

```diff
@@ -211,7 +204,7 @@ export class AuthContainer {
       await this.addAccount({
         name: form.name.$,
```

Comment:
`$` is the validate function provided by `https://formstate.github.io/#/`, which we used to deal with form. It will call the registered validators and return if passed the validation.